turbopy/diagnostics.py

```diff
@@ -162,12 +163,41 @@ def finalize(self):
 
 
 class FieldDiagnostic(Diagnostic):
+    """
+    Parameters
+    ----------
+    owner : Simulation
+       Simulation object containing current object.
+    input_data : dict
+       Dictionary that contains information regarding location, field, and output type.
+
+    Attributes
+    ----------
+    component : str
+    field_name : str
+        Field.
+    output : str
+        Output type.
+    field : str, None
+        Field as dictated by resource.
+    dump_interval : SimulationClock, None
+        Time interval between two diagnostic runs.
+    last_dump : SimulationClock, None
+        Time of last diagnostic run.
+    diagnose : method
+        Run do_diagnostic method.
```

Comment:
Not quite. If a dump interval is specified, then `diagnose` will be changed to point to `check_step`. Maybe this could say:
```suggestion
        Run `do_diagnostic` or `check_step` method depending on configuration parameters.
```